qemu/tests/win_heavyload.py

```diff
@@ -139,9 +139,16 @@ def add_option(cmd, key, val):
             if not installed and download_url:
                 utils.system("rm -f %s/HeavyLoad*.exe" % tmp_dir)
                 session.cmd("del /f /s %sHeavyLoad*.exe" % dst)
-            if loop_session_cmd(session, check_running_cmd):
-                if not loop_session_cmd(session, stop_cmd):
-                    raise error.TestFail("Unable to terminate heavyload "
-                                         "process")
+            for i in range(3):
```

Comment:
1. If use loop_session_cmd directly, it need be modified, otherwise it will cause the bz that this patch want to fix.
2. repeate 3 times is a old design, to avoid can not stop heavyload at one time. 
Agree with you, this is a clean step, won't effect the test.  In fact, i think all of these can de deleted before, as there has "session.close" at last,  do you think it is acceptable to delete this step?